packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -539,10 +556,44 @@ function LineItemSublabelAndPrice( {
 	}
 
 	if (
-		( isDomainRegistration || isDomainMap || isGSuite ) &&
-		product.months_per_bill_period === 12
+		isGoogleWorkspaceProductSlug( productSlug ) ||
```

Comment:
It might be worth it to extract this logic into own function since we use it multiple times in this PR.

Reply:
That's a good point. I've refrained from doing so mostly because we already have such a helper in the backend as `WPCOM_Store::is_email_product()` which doesn't handle extra licenses for G Suite: I'm worried of introducing a similar function in Calypso with a (slightly) different behavior.